packages/kit/src/packaging/index.js

```diff
@@ -144,11 +153,44 @@ export async function make_package(config, cwd = process.cwd()) {
 		const full_path = path.join(cwd, pathname);
 		if (fs.lstatSync(full_path).isDirectory()) continue; // just to be sure
 
-		const package_path = path.join(cwd, config.kit.package.dir, pathname);
+		const package_path = path.join(abs_package_dir, pathname);
 		if (!fs.existsSync(package_path)) fs.copyFileSync(full_path, package_path);
 	}
 }
 
+/**
+ * Resolves the `$lib` alias.
+ *
+ * TODO: make this more generic to also handle
+ * other aliases the user could have defined via `kit.vite.resolve.alias`.
+ *
+ * @param {string} file Relative to the lib root
+ * @param {string} content
+ * @param {import('types/config').ValidatedConfig} config
+ * @returns {string}
+ */
+function resolve_$lib_alias(file, content, config) {
+	/**
+	 * @param {string} match
+	 * @param {string} _
+	 * @param {string} import_path
+	 */
+	const replace_import_path = (match, _, import_path) => {
+		if (!import_path.startsWith('$lib/')) {
+			return match;
+		}
+
+		const full_path = path.join(config.kit.files.lib, file);
+		const full_import_path = path.join(config.kit.files.lib, import_path.slice('$lib/'.length));
+		let resolved = path.relative(path.dirname(full_path), full_import_path).replace(/\\/g, '/');
```

Comment:
What happens when `full_import_path` doesn't actually exists, for whatever reasons it may be (most likely a bad copy-paste or refactor)?

`path.relative` won't throw as long as it's a string, should we throw an error, leave it with the original alias in place, or use the non-existent resolved path (currently the case)?

Reply:
I think that's on the user then, the also things like `svelte-check` should be able find out. We don't check other file paths for correctness, so in my mind it would be inconsistent to check the aliases.